### PR TITLE
Add Of Modify Social Media Permission

### PR DESCRIPTION
**Pull request by GuenoleCHERUBIN**
Added the form to modify the permissions on the social medias

### CODE REVIEW

File: controllers/ManagementController.php [hunk 1]
```diff
@@ -101,6 +131,18 @@ private function checkIfUserExist($id)
         throw new $unauthorizedException();
     }
 
+    /**
+     * @param int $id
+     * @return SocialMediaPermission|null
+     */
+    private function chekcIfUserPermission($id)
```
Comment (EmileCalixte) [inline controllers/ManagementController.php]: This function name is incorrect: I think we should name it "checkIfPermissionExists" to be consistent with checkIfUserExist function

File: controllers/ManagementController.php [hunk 2]
```diff
@@ -45,23 +47,36 @@ public function actionAccounts($id = null)
...
+        $formModifyAccount->is_active    = $user->is_active;
+
+        $formSocialMediaPermission->facebook_enabled = $userPermission->facebook_enabled;
+        $formSocialMediaPermission->twitter_enabled  = $userPermission->twitter_enabled;
+        $formSocialMediaPermission->linkedin_enabled = $userPermission->linkedin_enabled;
 
         return $this->render('modifyAccount', [
-            'model'=>$form,
+            'formModifyAccount_model'=>$formModifyAccount,
+            'formSocialMediaPermission_model'=> $formSocialMediaPermission,
```
Comment (EmileCalixte) [inline controllers/ManagementController.php]: Please add spaces around "=>"

**Review by EmileCalixte — CHANGES_REQUESTED** [review]

File: controllers/ManagementController.php (hunk 1 above)
Comment (GuenoleCHERUBIN) [reply controllers/ManagementController.php]: You're right

**Review by EmileCalixte — APPROVED** [review]

File: views/management/modifyAccount.php [hunk 3]
```diff
@@ -12,38 +13,68 @@
...
+                    'class' => 'btn btn-primary', 'name' => 'modify-account-button'
+            ]) ?>
         </div>
     </div>
 
 
-    <?php ActiveForm::end(); ?>
+    <?php ActiveForm::end();
+
+    if (in_array($formModifyAccount_model->role, ['publisher','admin'])) {
```
Comment (alexisljn) [inline views/management/modifyAccount.php]: Why do you want to display the social media permissions form only if the edited user is a publisher or an admin ? I think that you want to make the very opposite and you must add "!" before "in_array". In this case it means that an admin cannot remove the publisher status of a user, it's not really what we need. You have to reconsider your approach

File: views/management/modifyAccount.php [hunk 4]
```diff
@@ -12,38 +13,68 @@
...
+        $formModifySocialMediaPermission = ActiveForm::begin([
+            'id' => 'modifySocialMedia-form',
```
Comment (alexisljn) [inline views/management/modifyAccount.php:54]: You start with camel case then end with kebab case

**Pull request merged**